ts/features/services/home/components/FeaturedServiceCard.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  Avatar,
+  Badge,
+  H4,
+  IOColors,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import I18n from "../../../../i18n";
+import { WithTestID } from "../../../../types/WithTestID";
+import { logoForService } from "../utils";
+import OrganizationNameLabel from "./OrganizationNameLabel";
+
+export type FeaturedServiceCardProps = WithTestID<{
+  id: string;
+  name: string;
+  organization_name?: string;
```

Comment:
Writing convention for props is `camelCase`:
```suggestion
  organizationName?: string;
```

Reply:
Sure, it's my typo/mistake deriving from actual naming for api response properties.
fe840b861